### DISCUSSION TITLE
Help with running the dewhale component generation

### DISCUSSION THREAD

**Discussion by Poorna-Athreya** [Yuyz0112/dewhale#329] [Q&A]
Hello! I am currently facing this error as shown below when I try to create a landing page component using the react generation library with the following prompt: "Create a landing page for an Ecommerce site called "Dungeon crawl" that allows users to buy items from a variety of categories"
I have added a WHITELIST and a OPENAI_API_KEY repo secret as suggested in this setup document. I am using an Azure open AI key and have also added an OPENAI_API_BASE_URL. Please help troubleshoot this error!
<img width="767" alt="Screenshot 2024-06-13 at 1 12 42 PM" src="https://github.com/Yuyz0112/dewhale/assets/99252985/dfcabaec-d532-4055-8b2c-3a00b2164846">

**Comment by arvindanta** [comment]
@Poorna-Athreya did u a find a solution for this?

**Comment by papipopi12** [comment]
Hey , are you  intrested about meme coin , if you let me know!
